turbinia/evidence.py

```diff
@@ -711,7 +786,7 @@ def _postprocess(self):
         self.state[EvidenceState.ATTACHED] = False
 
 
-class GoogleCloudDisk(RawDisk):
+class GoogleCloudDisk(Evidence):
```

Comment:
This class still uses `device_path`, so we should either still subclass RawDisk or add `device_path` as an attribute explicitly.

Reply:
I think adding ```device_path``` as an attribute makes more sense in this case.

Reply:
Done